cmd/create-index-snapshot/main.go

```diff
@@ -54,7 +54,7 @@ func run() int {
 	pflag.Parse()
 
 	// Initialize the logger.
-	zerolog.TimestampFunc = func() time.Time { return time.Now() }
+	zerolog.TimestampFunc = time.Now
```

Comment:
These are both from. It should be `func() time.Time { return time.Now().UTC() }`. It's the cleanest way to get Zerolog to log in UTC time.